src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -275,7 +297,7 @@ def json(self, path: str) -> DataFrame:
         """Specify the path of the JSON file(s) to load.
 
         Args:
-            path: The stage location of a JSON file, or a stage location that has JSON files.
+            path: The stage location of a JSON file, or location of a JSON file.
```

Comment:
hmm I meant that similar to the sentence for csv, you should have
```
path: A path pointing to a local JSON file, or a JSON file on a stage.
```
Same below. Sorry for not being clear.